.circleci/config.yml

```diff
@@ -59,8 +59,12 @@ jobs:
           command: |
             . venv/bin/activate
             cd python
-            python3 -m pytest ml4ir/applications/classification/tests --junitxml=test_results/results.xml
+            COVERAGE_FILE=reports/.coverage.classif coverage run  --include "ml4ir/*" -m pytest ml4ir/applications/classification/tests/ --junitxml=test_results/results.xml
           no_output_timeout: 45m  # Time for running the tests before we timeout
+      - persist_to_workspace:
```

Comment:
This is used to share data between builds (https://circleci.com/docs/2.0/workflows/#using-workspaces-to-share-data-among-jobs)